hivemind/moe/server/server.py

```diff
@@ -189,21 +189,26 @@ def create(
         else:
             args_schema = (BatchTensorDescriptor.from_tensor(sample_input, compression),)
 
-        scheduler = schedule_name_to_scheduler[scheduler]
+        scheduler_cls = schedule_name_to_scheduler[scheduler]
+        if scheduler_cls is not None:
```

Comment:
Perhaps it’s better to check `scheduler` for None even before accessing the dictionary

Reply:
scheduler is a string, it cannot be None. It can, though, be a sting "none", but that is handled by the dictionary